roundrect.xcodeproj/xcshareddata/xcschemes/roundrect.xcscheme

```diff
@@ -26,6 +26,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      codeCoverageEnabled = "YES"
```

Comment:
Not related to the current change set, but I'll allow it 👨‍⚖️

Reply:
This and the deleted code could've all been done elsewhere